### DISCUSSION TITLE
Start from the count matrix tutorial no longer available

### DISCUSSION THREAD

**Discussion by degrainger** [aertslab/pycisTopic#145] [General]
Hello! I was looking at the best way to run SCENIC+ on my Seurat/Signac analysed 10x multiome data. I saw in the FAQ for pycisTopic there is a link to a tutorial on how best to do this but the link no longer works. Please could you update it? Thank you!

https://pycistopic.readthedocs.io/en/latest/faqs.html#i-have-a-cistopic-analysis-in-r-can-i-still-use-pycistopic-scenic

**Comment by SeppeDeWinter** [comment]
Hi @degrainger 

Please find this notebook here: https://github.com/aertslab/pycisTopic/blob/old/notebooks/Toy_melanoma-RTD.ipynb

Seppe
